app/konnect/network.md

```diff
@@ -97,4 +98,8 @@ You can find the Telemetry hostname through the Runtime Manager:
 {% endnavtabs %}
 
 {:.note}
-> **Note**: Visit [https://ip-addresses.origin.konghq.com/ip-addresses.json](https://ip-addresses.origin.konghq.com/ip-addresses.json) for the list of IPs associated to regional hostnames. You can also subscribe to [https://ip-addresses.origin.konghq.com/rss](https://ip-addresses.origin.konghq.com/rss) for updates.  
+> **Note**: Visit [https://ip-addresses.origin.konghq.com/ip-addresses.json](https://ip-addresses.origin.konghq.com/ip-addresses.json) for the list of IPs associated to regional hostnames. You can also subscribe to [https://ip-addresses.origin.konghq.com/rss](https://ip-addresses.origin.konghq.com/rss) for updates. 
+
+### Mesh Manager
+
+If you plan to use [Mesh Manager](/konnect/mesh-manager/) to manage your Kong service mesh, you must add the `<region>.mesh.sync.konghq.com:443` hostname to your firewall allowlist. 
```

Comment:
Do we want to re-iterate the possible options for `<region>` here? E.g. `Region can be us, eu, or global.` in the same way that it's called out above for Gateway Manager.